analysis/src/HistoManager.cxx

```diff
@@ -104,11 +110,15 @@ void HistoManager::DefineHistos(std::vector<std::string> histoCopyNames, std::st
                 h_name = m_name+"_"+ histoCopyNames.at(i) + "_" + hist.key() ;
                 singleCopy = false;
             }
-
-            std::cout << "DefineHisto: " << h_name << std::endl;
-            std::size_t found = (hist.key()).find_last_of("_");
-            std::string extension = hist.key().substr(found+1);
-
+            if(debug_){
+                std::cout << "DefineHisto: " << h_name << std::endl;
+            }
+                std::size_t found = (hist.key()).find_last_of("_");
+                std::string extension = hist.key().substr(found+1);
+                std::string xtitty = hist.value().at("xtitle");
```

Comment:
xtitty??

Reply:
since "extension" and "xtitty" are only used here in the debug, maybe we should not make variables for them and just call these in the std::cout stream line.

Reply:
extension is not just used for debug; it also sets up whether we populate a 1 or 2d histogram. title may be only used for debug. So it has been removed